index.ts

```diff
@@ -49,6 +50,10 @@ interface IlpForwardedPayment {
 }
 
 const serializeIlpPayment = (json: IlpPayment) => {
+  assert(json.amount && typeof json.amount === 'string', 'amount must be a string')
+  assert(typeof json.account === 'string', 'account is required')
+  assert(!json.data || Buffer.isBuffer(json.data), 'data must be a buffer')
```

Comment:
Is the `!json.data` needed? Just curious, I would have assumed `Buffer.isBuffer` returns false for anything falsy passed into it.

Reply:
`data` is optional so this is just checking that it's a buffer if it exists